actions/admin/settings/180.dkim.php

```diff
@@ -36,25 +36,25 @@
 					'varname' => 'dkim_prefix',
 					'type' => 'string',
 					'string_type' => 'dir',
-					'default' => '/etc/postfix/dkim/',
+					'default' => '/etc/dkim-froxlor/',
 					'save_method' => 'storeSettingField'
 				),
-				'dkim_domains' => array(
+/*				'dkim_domains' => array(
 					'label' => $lng['dkim']['dkim_domains'],
 					'settinggroup' => 'dkim',
 					'varname' => 'dkim_domains',
 					'type' => 'string',
 					'string_regexp' => '/^[a-z0-9\._]+$/i',
 					'default' => 'domains',
 					'save_method' => 'storeSettingField'
-				),
+				), */
 				'dkim_dkimkeys' => array(
 					'label' => $lng['dkim']['dkim_dkimkeys'],
 					'settinggroup' => 'dkim',
 					'varname' => 'dkim_dkimkeys',
 					'type' => 'string',
-					'string_regexp' => '/^[a-z0-9\._]+$/i',
-					'default' => 'dkim-keys.conf',
+					'string_type' => 'dir',				// DF8OE
```

Comment:
no need for the `// DF8OE` comment everywhere

Reply:
This waqs because I started on 0.9 installation - I will remove all these tags.